.jenkins/pytorch/common.sh

```diff
@@ -74,7 +74,7 @@ if [[ "$BUILD_ENVIRONMENT" != *win-* ]]; then
   if which sccache > /dev/null; then
     # Save sccache logs to file
     sccache --stop-server > /dev/null  2>&1 || true
-    rm ~/sccache_error.log || true
+    rm -f ~/sccache_error.log
```

Comment:
Please keep `|| true` in place, as we do not want to error out if log can not be deleted
```suggestion
    rm -f ~/sccache_error.log || true
```